.github/workflows/simorgh-local-server-tests.yml

```diff
@@ -41,3 +41,8 @@ jobs:
 
       - name: Run Lighthouse
         run: npm run lighthouse
+
+      - name: Run Puppeteer Tests
+        run: | 
+          npm install --no-save puppeteer
```

Comment:
is there anyway we can use `npx` here instead of installing puppeteer and then running it?

Reply:
I'll give it a shot!

Reply:
Mmm not sure if it's possible, because the test:puppeteer command in Simorgh just runs jest -> https://github.com/bbc/simorgh/blob/latest/package.json#L41

Running it takes 2 minutes, so it's not much of an overhead

Reply:
No worries, just wanted to see if it would work